{
  "comments": [
    {
      "key": {
        "uuid": "30ad4939_928483ba",
        "filename": "scripts/geopmpy/io.py",
        "patchSetId": 1
      },
      "lineNbr": 117,
      "author": {
        "id": 1012567
      },
      "writtenOn": "2019-08-27T00:15:00Z",
      "side": 1,
      "message": "I don\u0027t remember exactly what we wanted with this, but it looks like a hack just to get it working.  In practice we rarely combine reports from different folders.",
      "range": {
        "startLine": 117,
        "startChar": 16,
        "endLine": 117,
        "endChar": 59
      },
      "revId": "75e10fe85ecc6bb9e03e6e9f08f80251ccf4ff7a",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "24680f10_cd2d5ffa",
        "filename": "scripts/test/TestIO.py",
        "patchSetId": 1
      },
      "lineNbr": 467,
      "author": {
        "id": 1009284
      },
      "writtenOn": "2019-08-19T23:44:09Z",
      "side": 1,
      "message": "To me, this indicates a bug in the parser and is a valid failure that should not be worked around.",
      "range": {
        "startLine": 465,
        "startChar": 8,
        "endLine": 467,
        "endChar": 38
      },
      "revId": "75e10fe85ecc6bb9e03e6e9f08f80251ccf4ff7a",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "a5a838b3_f7b0d35e",
        "filename": "scripts/test/TestIO.py",
        "patchSetId": 1
      },
      "lineNbr": 467,
      "author": {
        "id": 1016629
      },
      "writtenOn": "2019-08-20T18:31:40Z",
      "side": 1,
      "message": "Agreed. I\u0027ve added a note about this to https://github.com/geopm/geopm/issues/698",
      "parentUuid": "24680f10_cd2d5ffa",
      "range": {
        "startLine": 465,
        "startChar": 8,
        "endLine": 467,
        "endChar": 38
      },
      "revId": "75e10fe85ecc6bb9e03e6e9f08f80251ccf4ff7a",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "fe4c1b98_503b8194",
        "filename": "scripts/test/TestIO.py",
        "patchSetId": 1
      },
      "lineNbr": 467,
      "author": {
        "id": 1016629
      },
      "writtenOn": "2019-08-21T22:23:49Z",
      "side": 1,
      "message": "Ended up just adding the fix to this diff. Turns out there\u0027s already a pattern for other fields with similar issues.",
      "parentUuid": "a5a838b3_f7b0d35e",
      "range": {
        "startLine": 465,
        "startChar": 8,
        "endLine": 467,
        "endChar": 38
      },
      "tag": "autogenerated:gerrit:newPatchSet",
      "revId": "75e10fe85ecc6bb9e03e6e9f08f80251ccf4ff7a",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "a303cea9_a4511d3a",
        "filename": "scripts/test/TestIO.py",
        "patchSetId": 1
      },
      "lineNbr": 468,
      "author": {
        "id": 1009284
      },
      "writtenOn": "2019-08-21T00:19:11Z",
      "side": 1,
      "message": "?",
      "range": {
        "startLine": 468,
        "startChar": 8,
        "endLine": 468,
        "endChar": 46
      },
      "revId": "75e10fe85ecc6bb9e03e6e9f08f80251ccf4ff7a",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "f4268996_e7b3839e",
        "filename": "scripts/test/TestIO.py",
        "patchSetId": 1
      },
      "lineNbr": 468,
      "author": {
        "id": 1016629
      },
      "writtenOn": "2019-08-21T22:23:49Z",
      "side": 1,
      "message": "Done",
      "parentUuid": "a303cea9_a4511d3a",
      "range": {
        "startLine": 468,
        "startChar": 8,
        "endLine": 468,
        "endChar": 46
      },
      "tag": "autogenerated:gerrit:newPatchSet",
      "revId": "75e10fe85ecc6bb9e03e6e9f08f80251ccf4ff7a",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    }
  ]
}